README.md

```diff
@@ -1 +1,113 @@
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Setup mobile first] <a name="about-project"></a>
+
+## 🛠️ Built With html and CSS .This is the Aliza Amir Portfolio, that allows you understand what are my skills as a developer.
+
+### Tech Stack <a name="tech-stack"></a>
+Html
+CSS
+Visual studio code
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- *It has linters*
+- *It has MIT license*
+- *It has html*
+- *It has CSS*
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="Setup-mobile-first"></a>
+  I don't have live demo link currently.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+In order to run this project you need:
+
+Step 1: if you don't have git installed on your machine you can download it from [here](https://git-scm.com/downloads).
+
+Once you have git installed on your machine you can clone your project by running the command below to clone your project to your local machine
+
+
+`git clone https://github.com/alizaamir123/mobile-version
+
+Alternatively, you can download the zip file of your project by clicking on the `Code` button on the right side of your project page and clicking on `Download ZIP`
+
+Step 2: Locate the folder where you cloned your project to and open the `index.html` file in your browser to view your project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Author <a name="authors"></a>
+
+👤 *Aliza Amir*
+
+- GitHub: [@Alizaamir123](https://github.com/Alizaamir123)
+- LinkedIn: (https://www.linkedin.com/in/aliza-amir-545196215/)
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+- *CSS framework designing*
+- *More detailed*
+- *Master HTML*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+*please show your support to improve my skills .this will help me to work hard give a ⭐️ if you loke this project*
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+"Special thanks to "Muneeb Rehman" for helping me in this project. His dedication, skills, and collaborative spirit were essential to my success."
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [Optional]  Kindly add the correct path of the MIT LICENSE in your project, as I can see you have not added the path correctly the link not working . Please look at the below suggestion to solve the issues.
![Correct mit license readme](https://user-images.githubusercontent.com/92458236/235299563-22c7bc50-f41f-4522-9556-e66f7bdba9bc.jpg)